Best Practices/Challenges/UnitTests/UnitTests.Challenge/Interfaces/IDataService.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace UnitTests.Challenge.Interfaces
+{
+    public interface IDataService
```

Comment:
issue (blocking): Can we swap to use an Order object instead of a string, so we can include a test that a service will update a property before saving.